### DISCUSSION TITLE
Achievements public beta

### DISCUSSION THREAD

**Discussion by NhanAZ** [community/community#18156] [Other Feature Feedback, Questions, & Ideas] [answered]
Badges are given and displayed incorrectly, lots of pull requests are ignored...

**Comment by bailey** [comment]
Hi @NhanAZ, only merged pull requests will count toward Pull Shark. Could you share details on where you're seeing this problem?

**Reply by ancarda** [reply]
My profile [only lists 2 pull requests](https://github.com/ancarda?tab=achievements) on the achievement: https://github.com/dag/vlasisku/pull/7 and https://github.com/Seldaek/jsonlint/pull/75 -- I don't know why it's these two... is it random?

I have 40 merged pull requests: https://github.com/pulls?q=is%3Apr+author%3Aancarda+is%3Amerged -- should I not see "x40" for the achievement? If not, why?

**EDIT**: Some of these are in private repos. I didn't realize that link won't work for anyone else. Some public, merged PRs include https://github.com/pygments/pygments/pull/2054, https://github.com/OndraM/ci-detector/pull/101, and https://github.com/mumble-voip/mumble/pull/4888

**Reply by NhanAZ** [reply]
@bailey Your profile is an example.
I see you replied to 13 discussions but your Galaxite Brain badge is 2

**Comment by bsantosh909** [comment, marked as answer]
@NhanAZ I don't think pull requests are ignored.

The badge is showing only the **2nd** and **16th** pull request of yours.
Which accounts to 1st and 2nd level of the achievements.

Once you hit the 3rd level count, your achievement badge will update accordingly. ✌️

**Reply by NhanAZ** [reply]
Oh. I didn't notice that...
Download why is it 2nd and 16th?

**Reply by venthur** [reply]
What are the other levels and when do you get them?

**Reply by jagerman** [reply]
I see 2, 16, 128 and 1024; if the pattern holds it appears that each level is 8x the previous level.

**Reply by Explosion-Scratch** [reply]
Huh, very interesting, why x8, why not x2?

**Reply by ArkaprabhaChakraborty** [reply]
Ahh that explains

**Reply by drpaneas** [reply]
is there any API call to find out my current PR merge number?

**Reply by PAYCELLCART** [reply]
#20774
